src/Plugins/RunawayProcessKiller/RunawayProcessKillerExtension.cs

```diff
@@ -113,11 +116,12 @@ public override void OnWrapperStarted()
             else
             {
                 Logger.Warn("The process " + pid + " has no " + expectedEnvVarName + " environment variable defined. " 
-                    + "The process has not been started by this service, hence it won't be terminated.");
+                    + "The process has not been started by WinSW, hence it won't be terminated.");
```

Comment:
WinSW is not for Jenkins. It is a standalone project, which is being widely used outside. But I agree the message is not ideal. The intention was to say that "The process has not been created by any of winsw-powered services you may have on this machine", but maybe it is better to revert the change